rabbitmq/sentiment_analysis/version.md

```diff
@@ -1,5 +1,18 @@
 # Version History
 
+
+### Version 0.1.5 (2023-08-10)
+- Update email function to allow flexible SMTP authentication options
+
+### Version 0.1.4 (2023-07-01)
+- Renamed Minio related environment variables
+
 ### Version 0.1.3 (2023-05-01)
 - Fix sentiment debias model by copying in the trained model numberbatch-en-17.04b.txt
 - Fix wordnet cannot zip issue in dockerfile
+
+### Version 0.1.2 (2023-05-01)
+- ?
```

Comment:
@ywkim312 i could figure out what's the image about. you pushed that one 3 months ago. Could you try see if you have any clue?

Reply:
I believe that it was minio url change